src/libaktualizr/bootloader/bootloader.cc

```diff
@@ -75,6 +75,9 @@ void Bootloader::updateNotify() const {
       if (Utils::shell("fw_setenv upgrade_available 1", &sink) != 0) {
         LOG_WARNING << "Failed setting upgrade_available for u-boot";
       }
+      if (Utils::shell("fw_setenv bootupgrade_available 1", &sink) != 0) {
```

Comment:
@ricardosalveti Do we support two types,  `uboot_masked` and `fiovb`?

Reply:
@mike-sul We now support both